docs/iot/getting-started/artik-no-sd-4.0.md

```diff
@@ -263,21 +263,22 @@ For detailed information, see <https://wiki.tizen.org/Booting_Tizen_From_eMMC_On
 
 1.  Connect Smart Development Bridge (SDB) as described in the previous section.
 2.  Install the connectivity drivers:
-        1.  Download the plugin zip file from the **ARTIK530(4.0) Plugin** section at <http://developer.samsung.com/tizendevice/firmware> and follow the instructions.
 
-        ![ARTIK530(4.0) Plugin section](media/wifi_driver_artik530.png)
+    1.  Download the plugin zip file from the **ARTIK530(4.0) Plugin** section at <http://developer.samsung.com/tizendevice/firmware> and follow the instructions.
 
-        2.  For the case of the Linux shell (Linux), run the `sh` script given in the instructions. For example:
+    ![ARTIK530(4.0) Plugin section](media/wifi_driver_artik530.png)
```

Comment:
Would you please add more spaces here for the indentation as on artik-no-sd-5.0.md?

Reply:
getting-started/artik-no-sd-4.0.md has been removed from b900d64 commit.